lib/mixlib/install/generator/powershell/scripts/install_project.ps1

```diff
@@ -102,8 +104,19 @@ function Install-Project {
 }
 set-alias install -value Install-Project
 
-Function Install-ChefMsi($msi) {
-  $p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi" -Passthru -Wait
+Function Install-ChefMsi($msi, $addlocal) {
+  if ($addlocal -eq "service") {
+    $p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi ADDLOCAL=`"ChefServiceFeature`"" -Passthru -Wait
```

Comment:
@wrightp , tried with the following code:
```
$addlocal_str = @{"service" = "ADDLOCAL=```"ChefServiceFeature```""; "task" = "ADDLOCAL=```"ChefSchTaskFeature```""};

  if ($addlocal -eq "auto") {
    $p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi" -Passthru -Wait
  }
  Else {
    $p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi $addlocal_str[$addlocal]" -Passthru -Wait
  }
```

Adding double escape for the grave returns the expected output on console as `ADDLOCAL=`"ChefServiceFeature`"`, but it still fails with `Start-Process` command with 1619 Exitcode.

You can notice that it's not possible to completely avoid the `If-Else` condition.